variables.tf

```diff
@@ -49,6 +49,26 @@ variable "auth_settings" {
   default     = null
 }
 
+variable "auth_settings_v2" {
```

Comment:
Since `auth_settings_v2` is upward compatible with `auth_settings`, it should be integrated into `auth_settings` as a configuration value.

Reply:
Not a problem, I’m travelling at the moment so I may not get to this until Wednesday.

Would you like me to replace the legacy auth_settings completely for simplicity or try to maintain comparability with both?

Reply:
I see no problem in the form of replacing the old `auth_settings`. If some incompatibilities arise, they will be handled by versioning.

Reply:
@shibayan `auth_settings` replaced, let me know if you're happy with this or want further changes.